frontend/smartComponents/SmartGameAbilitiesList.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { fetchGameAbilitiesListData } from '../storage/utils/fetchGameData';
+import { GameAbilitiesListProps, GameAbilitiesList } from '../components/GameAbilitiesList';
+
+interface SmartGameAbilitiesListState {
+	isDataAvailable: boolean;
+	gameAbilitiesListProps: GameAbilitiesListProps | null;
+}
+
+interface SmartGameAbilitiesListProps {
+	gameId: number;
+}
+
+export class SmartGameAbilitiesList extends React.Component<SmartGameAbilitiesListProps, SmartGameAbilitiesListState> {
+	constructor(props: SmartGameAbilitiesListProps) {
+		super(props);
+		this.state = {
+			isDataAvailable: false,
+			gameAbilitiesListProps: null,
+		};
+	}
+	public componentDidMount() {
+		fetchGameAbilitiesListData(this.props.gameId)
```

Comment:
You can make `componentDidMount` `async` to make the code flatter: https://www.valentinog.com/blog/await-react/